Epam.TestAutomation.Tests.BDD/Epam.TestAutomation.Tests.BDD.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Remove BDD project from data_driven_testing branch as it is not a part of DDT homework.